airbyte-cdk/python/airbyte_cdk/sources/declarative/partition_routers/substream_partition_router.py

```diff
@@ -148,7 +154,9 @@ def stream_slices(self) -> Iterable[StreamSlice]:
                             pass
                         else:
                             empty_parent_slice = False
-                            yield {stream_state_field: stream_state_value, "parent_slice": parent_slice}
+                            yield PerPartitionStreamSlice(
+                                partition={stream_state_field: stream_state_value, "parent_slice": parent_partition}, cursor_slice={}
```

Comment:
Can we updated the naming for `stream_state_field` and `stream_state_value`? With this change, it becomes apparent that it's not related to the state, right?